java/src/main/native/src/Aggregation128UtilsJni.cpp

```diff
@@ -24,26 +24,28 @@ JNIEXPORT jlong JNICALL Java_ai_rapids_cudf_Aggregation128Utils_extractInt32Chun
   JNIEnv* env, jclass, jlong j_column_view, jint j_out_dtype, jint j_chunk_idx)
 {
   JNI_NULL_CHECK(env, j_column_view, "column is null", 0);
-  try {
+  JNI_TRY
+  {
```

Comment:
usually we had `try { ` in the past. Why not make this `JNI_TRY {` instead of adding the newline?

Reply:
ok, that's fine.